src/views/charts/LaunchableMarketChart.tsx

```diff
@@ -143,6 +145,20 @@ export const LaunchableMarketChart = ({
     );
   };
 
+  const trackTimeframeChange = useCallback(
+    (_timeframe: string) => {
```

Comment:
why _ before the name?

Reply:
Ah I'll update this to `selectedTimeframe`. Originally I just didn't want it to clash with `timeframe` from `useState` regardless of scope